cli/js/lib.deno.ns.d.ts

```diff
@@ -71,6 +71,12 @@ declare namespace Deno {
   /** Exit the Deno process with optional exit code. */
   export function exit(code?: number): never;
 
+  /** Get process priority */
```

Comment:
The more I read the Deno API, the more I am appreciating thorough documentation.  I quite like the Node API documentation as well and I wonder if we should pinch their more verbose description of this (assuming it is accurate of course!)?

> Returns the scheduling priority for the process specified by pid. If pid is not provided or is 0, the priority of the current process is returned.

Reply:
i aggree will do.